selfdrive/car/body/values.py

```diff
@@ -16,23 +17,35 @@ class CarControllerParams:
 
 class CAR:
   BODY = "COMMA BODY"
+  BODY_KNEE = "COMMA BODY WITH KNEE"
 
 CAR_INFO: Dict[str, CarInfo] = {
   CAR.BODY: CarInfo("comma body", package="All", harness=Harness.none),
+  CAR.BODY_KNEE: CarInfo("comma body + knee", package="All", harness=Harness.none),
 }
 
 FW_VERSIONS = {
   CAR.BODY: {
     (Ecu.engine, 0x720, None): [
-      b'0.0.01',
-      b'02/27/2022'
+      # b'0.0.02',
+      b'ELECTRIC0'
```

Comment:
I think we'll want to finish auto fw update before merging, or at least before release